apps/condo/pages/ticket/index.tsx

```diff
@@ -53,14 +54,96 @@ const TAP_BAR_ROW_GUTTER: [Gutter, Gutter] = [0, 20]
 const CHECKBOX_STYLE: CSSProperties = { paddingLeft: '0px', fontSize: fontSizes.content }
 const TOP_BAR_FIRST_COLUMN_GUTTER: [Gutter, Gutter] = [40, 20]
 
+const TICKETS_RE_FETCH_INTERVAL = 5 * 1000
+
+const TicketsTable = ({ filterMetas, sortBy, searchTicketsQuery }) => {
+    const router = useRouter()
+
+    const { filters, offset } = parseQuery(router.query)
+    const currentPageIndex = getPageIndexFromOffset(offset, DEFAULT_PAGE_SIZE)
+
+    const {
+        loading: isTicketsFetching,
+        count: total,
+        objs: tickets,
+        refetch,
+    } = Ticket.useObjects({
+        sortBy,
+        where: searchTicketsQuery,
+        first: DEFAULT_PAGE_SIZE,
+        skip: (currentPageIndex - 1) * DEFAULT_PAGE_SIZE,
+    })
+
+    const { useFlag, updateContext } = useFeatureFlags()
+    const { organization } = useOrganization()
+
+    useEffect(() => {
+        updateContext({ organization: organization.id })
+    }, [organization, updateContext])
+
+    const isRefetchTicketsFeatureEnabled = useFlag(RE_FETCH_TICKETS_IN_CONTROL_ROOM)
+
+    const [isRefetching, setIsRefetching] = useState(false)
+    useEffect(() => {
+        if (isRefetchTicketsFeatureEnabled) {
+            const handler = setInterval(async () => {
+                setIsRefetching(true)
+                await refetch()
+                setIsRefetching(false)
+            }, TICKETS_RE_FETCH_INTERVAL)
+            return () => {
+                clearInterval(handler)
```

Comment:
will this work if `isRefetchTicketsFeatureEnabled` value changes from `true` to `false`?